cupy/core/core.pyx

```diff
@@ -567,18 +568,37 @@ cdef class ndarray:
         newarray._set_shape_and_strides(shape, strides, False, True)
         return newarray
 
-    def reshape(self, *shape):
+    def reshape(self, *shape, **kwargs):
```

Comment:
How about a named argument (`order='C'`) instead of `**kwargs`?
Cython supports the syntax even under Python 2.

Reply:
sure. I wasn't aware of that